### PR TITLE
Call `system_profiler` only once per timer intervall.

### PR DESCRIPTION
**Pull request by reitermarkus**

### CODE REVIEW

**Comment by duanefields** [comment]
Since I don't have the ability to test this can I get a thumbs up from a second contributor?

**Comment by SoneeJohn** [comment]
@duanefields  will test it out today.

**Comment by reitermarkus** [comment]
@SoneeJohn, did you already get around to testing this?

**Comment by SoneeJohn** [comment]
@reitermarkus Will test it in the next hour or so. Sorry for the delay 🙂

File: VirtualKVM/KVMThunderboltObserver.m [hunk 1]
```diff
@@ -1,13 +1,14 @@
 #import "KVMThunderboltObserver.h"
 #import "KVMSystemProfiler.h"
 
-static NSTimeInterval const kTimeInterval = 2.0;
+static NSTimeInterval const kTimeInterval = 2.5;
```
Comment (reitermarkus) [inline VirtualKVM/KVMThunderboltObserver.m]: No real reason. Will revert it if you think it's too long.

Comment (SoneeJohn) [inline VirtualKVM/KVMThunderboltObserver.m]: @reitermarkus I think you can revert it if you like.

**Comment by reitermarkus** [comment]
This should be ready to merge.

**Pull request merged**